docs/source/using/addons.adoc

```diff
@@ -112,6 +112,23 @@ The following table shows the supported variables that are available for interpo
 |Name of the current add-on.
 |===
 
+The commands xref:../command-ref/minishift_start.adoc#[`minishift start`] as well as xref:../command-ref/minishift_addons_apply.adoc#[`minishift addons apply`] also provide a `--addon-env` flag which allow to pass variables for interpolation dynamically, for example:
+
+----
+$ minishift addons apply --addon-env ACME_TOKEN=1234 acme
+----
+
+The `--addon-env` flag can be specified multiple times to define more than one variables for interpolation.
```

Comment:
> You should mention the short form as well i.e. -a .

I don't think this is required. We do this at no other point in the docs. Things just get cumbersome to describe and sentences unnecessarily complicated. People who want this option will check the command ref.

> Also we should mention about how it can take variables from environment

What do you mean? This would then again go into variable interpolation of the shell which is a topic outside of the scope of Minishift.